product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -70,11 +70,11 @@ performance.
 
 The `nprovisional_waits` value reflects the number of operations on the same
 tuples being performed by concurrent apply transactions. These are provisional
-waits that aren't actually waiting yet but could. 
+waits that aren't actually waiting yet but could.
```

Comment:
Can we clarify the conditional here? Like "...but could be if..."

Reply:
```suggestion
waits that aren't actually waiting yet but could start waiting.
```